test/k4FWCoreTest/src/components/ExampleParticleIDConsumer.cpp

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2014-2024 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <edm4hep/ReconstructedParticleCollection.h>
+#include "k4FWCore/Consumer.h"
+#include "k4FWCore/MetadataUtils.h"
+
+#include "edm4hep/ParticleIDCollection.h"
+#include "edm4hep/utils/ParticleIDUtils.h"
+
+#include <Gaudi/Property.h>
+
+#include "fmt/format.h"
+#include "fmt/ranges.h"
```

Comment:
```suggestion
```
Only `fmt::format` is being used

Reply:
No, that's a false positive (negative?) from llvm. `pidMeta.paramNames` is a `std::vector` which requires `fmt/ranges.h`.